autosynth/github.py

```diff
@@ -209,3 +361,33 @@ def get_api_label(self, repository: str, synth_path: str) -> Optional[str]:
                 return label
 
         return None
+
+
+def _handle_response_json(response: requests.Response) -> Union[Dict, List]:
```

Comment:
To help the reader understand the code that calls this function, let's give it a name that describes what it does.  How about `_get_json_or_raise_exception()`?

Reply:
Done.